### DISCUSSION TITLE
(intermediate value).default is not a function when running npx cypress-cloud

### DISCUSSION THREAD

**Discussion by bilarallen** [currents-dev/cypress-currents-reporter#69] [Q&A] [answered]
Hello, I was trying to use cypress-cloud while there's still an issue with the currents-cli. I followed the steps here https://github.com/currents-dev/cypress-cloud#setup for setting up the cypress-cloud but every time I run the command `npx cypress-cloud --parallel --record --key <your_key> --ci-build-id hello-cypress-cloud` it always returns `(intermediate value).default is not a function` in the CLI.

![image](https://user-images.githubusercontent.com/65072341/224236341-aaf41755-92bc-4e4e-97e4-eef405bfddc6.png)
![image](https://user-images.githubusercontent.com/65072341/224236512-c8b0e502-7a76-411b-9af6-d909e3f128ea.png)
![image](https://user-images.githubusercontent.com/65072341/224236546-1e40eaac-807b-423c-a1cf-1d35f8a1cf08.png)

Is this a bug or I am doing or encountering something wrong here? Hope for some help. Thanks!

**Comment by agoldis** [comment, marked as answer]
It was a bug in the published package. Please update to 1.4.2 and use 
```
const {cloudPlugin} = require("cypress-cloud/plugin");
```

**Reply by bilarallen** [reply]
Hi @agoldis I just install the updated version but the issue still exist 
![image](https://user-images.githubusercontent.com/65072341/224251902-c8a35211-a425-4280-8d05-655444c92bb6.png)

![image](https://user-images.githubusercontent.com/65072341/224252219-5f147117-83c9-428f-a32e-eafd521a1091.png)

**Comment by agoldis** [comment]
@bilarallen weird. Please submit the your environment information as appears in https://github.com/currents-dev/cypress-cloud/issues/new?assignees=&labels=bug&template=1.bug.yaml

![currents-2023-03-09-23 41 08@2x](https://user-images.githubusercontent.com/1637928/224253830-fa2e75e7-4595-4ea6-8332-f9e49cce1e63.png)

**Reply by bilarallen** [reply]
Hi @agoldis yep here's the issue https://github.com/currents-dev/cypress-cloud/issues/71 thanks!
